docs/source/cartopy.rst

```diff
@@ -27,23 +27,24 @@ The top level cartopy module contains the :attr:`~cartopy.config` dictionary whi
 
     Keys in the config dictionary:
 
-     * ``pre_existing_data_dir`` - the absolute path to a directory where standard
-                                   data (such as that from NaturalEarth) can be
-                                   found. If it is not found in this location
-                                   the ``data_dir`` config item will be used.
-
-     * ``data_dir`` - the absolute path to a directory where standard data (such
-                      as that from NaturalEarth) can be found. If it is not found
-                      and the item is downloadable cartopy will download the
-                      appropriate file(s) to a subdirectory of this directory,
-                      therefore ``data_dir`` should be writable by the user.
-
-     * ``repo_data_dir`` - the absolute path to the directory where the data
-                           delivered with the cartopy repository is stored.
-                           Typically this will only be set by OS packagers and
-                           system administrators for site wide deployments.
-
-     * ``downloaders`` - a dictionary mapping standard "specifications" to the
-                         appropriate :class:`~cartopy.io.Downloader`. For further
-                         documentation and an example see
-                         :func:`cartopy.io.Downloader.from_config`.
+    ``pre_existing_data_dir``
+        The absolute path to a directory where standard data (such as that from
+        NaturalEarth) can be found. If it is not found in this location the
+        ``data_dir`` config item will be used.
+
+    ``data_dir``
+        The absolute path to a directory where standard data (such as that from
+        NaturalEarth) can be found. If it is not found and the item is
+        downloadable cartopy will download the appropriate file(s) to a
+        subdirectory of this directory, therefore ``data_dir`` should be
+        writable by the user.
+
+    ``repo_data_dir``
+        The absolute path to the directory where the data delivered with the
+        cartopy repository is stored. Typically this will only be set by OS
+        packagers and system administrators for site wide deployments.
+
+    ``downloaders``
+        A dictionary mapping standard "specifications" to the appropriate
+        :class:`~cartopy.io.Downloader`. For further documentation and an
+        example see :func:`cartopy.io.Downloader.from_config`.
```

Comment:
Well, this format produces a definition list, which I think makes sense semantically. It may not have bullet points, that's more a problem of CSS formatting, I think.